src/Form/Config/Themes/Load.php

```diff
@@ -0,0 +1,5 @@
+<?php
+return [
+	require GIVE_PLUGIN_DIR . 'src/Form/Config/Themes/Sequola.php',
```

Comment:
@ravinderk note that it's spelt "Sequoia"

Reply:
fixed: https://github.com/impress-org/givewp/compare/a2b3dc9d537c...8b073f1d649f